src/Apps/Artwork/Components/ArtworkImageBrowser/UtilButton.tsx

```diff
@@ -126,7 +126,12 @@ const UtilButtonInnerText: React.FC<UtilButtonInnerTextProps> = ({
     return null
   }
 
-  if (longestLabel) {
+  /**
+   * We ignore this, since this functionality is not needed for tests and will add some problems
+   * e.g. `screen.getByText("Label")` will return error,
+   * since there will be 2 elements with "Label" text on the page.
+   */
+  if (longestLabel && process.env.NODE_ENV !== "test") {
```

Comment:
**question:** shouldn't we have this `process.env.NODE_ENV !== "test"` check only for the [`HiddenText`](https://github.com/artsy/force/pull/12132/files#diff-c2ea95d31134e0d393b03cd0dcfd9aa3a56f6689c59652f20b949a531ee932b1R138-R140) part of the component?

Reply:
I would be genuinely surprised if this didn't crash on the client?

Reply:
> I would be genuinely surprised if this didn't crash on the client?

We already use this in some components (like [here](https://github.com/artsy/force/blob/main/src/Components/Truncator.tsx#L32) and [here](https://github.com/artsy/force/blob/main/src/Apps/Order/Routes/Payment/PaymentContext/OrderPaymentContext.tsx#L99)). everything seems to be ok

Reply:
a good note about using `getENV`(I completely forgot about it). thanks again 🙏